### PR TITLE
Make console command work with url option

### PR DESCRIPTION
**Pull request by sietzekeuning**
To make the console command work for one specific URL this option is added. Now this package combined with laravel-remote becomes even more powerful!

### CODE REVIEW

File: src/Commands/ClearCommand.php [hunk 1]
```diff
@@ -9,15 +9,19 @@
...
+    protected $signature = 'responsecache:clear {--url=}';
 
     protected $description = 'Clear the response cache';
 
     public function handle(ResponseCacheRepository $cache)
     {
         event(new ClearingResponseCache());
 
-        $cache->clear();
+        if ($url = $this->option('url')) {
```
Comment (freekmurze) [inline src/Commands/ClearCommand.php:20]: Please use ternary to avoid that `else`

**Review by freekmurze — CHANGES_REQUESTED** [review]
Could you add a test to prove that this works?

**Comment by sietzekeuning** [comment]
Fixed the code and added an additional test to make sure only one page cache gets cleared.

File: tests/Commands/ClearCommandTest.php [hunk 2]
```diff
@@ -27,6 +27,26 @@ public function it_will_clear_the_cache()
         $this->assertDifferentResponse($firstResponse, $secondResponse);
     }
 
+    /** @test */
+    public function it_will_clear_only_one_page_from_cache()
+    {
+        $firstResponse            = $this->get('/random/1');
```
Comment (freekmurze) [inline tests/Commands/ClearCommandTest.php:33]: This alignment is not consistent with our code, please remove that.

**Comment by sietzekeuning** [comment]
Fixed alignment, sorry for my aggressive cs fixer :)

**Comment by freekmurze** [comment]
Do also remove that `else` 🙂

**Comment by sietzekeuning** [comment]
Removed the `else` statement  🙂

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**